content/en/docs/intro/CheatSheet.md

```diff
@@ -0,0 +1,124 @@
+# Cheat Sheet - HELM
+
+Helm cheatsheet featuring  all the necessary commands required to  manage application through Helm.
+
+-----------------------------------------------------------------------------------------------------------------------------------------------
+### Basic interpretations/context
+
+Chart:
+- It is name of your chart in case it has been pulled and untared.
```

Comment:
minor typo on this line: please change to 'untarred'

Reply:
Alright, will do that